lib/Conversion/TTIRToTTMetal/TTIRToTTMetal.cpp

```diff
@@ -850,7 +886,7 @@ class TTIRToTTMetalDispatchRewriter : public OpRewritePattern<ttir::GenericOp> {
 
     // For all unary ops first copy tile from input CB at inCBTileIndex to DST
     // register at dstTileIndex.
-    builder.create<ttkernel::CopyTileOp>(location, inCB, inCBTileIndex,
+    builder.create<ttkernel::CopyTileOp>(location, inCB, dstTileIndex,
```

Comment:
Is there somewhere that asserts the CB push/pop granularity is the same size as dst allocation?

Reply:
Yeah naming is a bit off at the moment. Compute kernels are working tile by tile, so we can always copy from index 0, `dstTileIndex` is not a proper name. I will fix it

Reply:
https://github.com/tenstorrent/tt-mlir/pull/918/commits/b2651dad7197a8bfc0883a1f35e3e91d8184f19b